coconut/src/tests/issuance_tests.rs

```diff
@@ -12,18 +12,23 @@ impl Coin {
         parameters: &Parameters,
         secret: &SecretKey,
         value: &Scalar,
+        seed: &Scalar,
         id: &Scalar,
     ) -> Self {
         let h0 = parameters.hs[0];
         let h1 = parameters.hs[1];
+        let h2 = parameters.hs[2];
         let o = Scalar::one();
-        let cm = h0 * value + h1 * id + parameters.g1 * o;
+        let cm = h0 * value + h1 * seed + h2 * id + parameters.g1 * o;
 
         let h = Parameters::hash_to_g1(cm.to_bytes());
 
         let y0 = &secret.ys[0];
         let y1 = &secret.ys[1];
-        Self(h, h * value * y0 + h * id * y1 + h * secret.x)
```

Comment:
If the order was (value, id) instead of (id, value) here before, shouldn't some test regarding value conservation be failing?

Reply:
I think it always is (value, id) otherwise tests would fail

Reply:
I thought we were switching the order somehow